gf/test/two_index_gf_test.cpp

```diff
@@ -262,13 +260,14 @@ TEST_F(TwoIndexGFTest,Assign)
     
     gf2=gf;
     EXPECT_EQ(data, gf2(omega,sigma));
-    EXPECT_THROW(other_gf=gf, std::invalid_argument);
+    EXPECT_THROW(other_gf = gf, std::invalid_argument);
     // EXPECT_EQ(data, other_gf(omega,sigma));
 }
 
 TEST_F(TwoIndexGFTest, DefaultConstructive)
 {
     gf_type gf_empty;
+#ifndef NDEBUG
```

Comment:
The same comment as https://github.com/ALPSCore/ALPSCore/pull/408/files#diff-f5308a091bc5c9a3c0d44b94de128649R256